### PR TITLE
[DOC-2231] feat(RBAC): add docs for query ownership privilege;

### PR DESCRIPTION
**Pull request by jue-yuan**
[DOC-2231](https://graphsql.atlassian.net/browse/DOC-2231)
Doc for RBAC: Fine-grained query privilege: New Query Privilege OWNERSHIP

[DOC-2231]: https://graphsql.atlassian.net/browse/DOC-2231?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by arun-TG-PM — APPROVED** [review]

File: modules/user-access/pages/fine-grained-query-privileges.adoc [hunk 1]
```diff
@@ -8,11 +8,25 @@ With fine-grained query privileges, following changes will take effects:
 
 * `WRITE_QUERY` privilege is deprecated. It has been split into privileges such as `CREATE_QUERY`, `UPDATE_QUERY` and `DROP_QUERY`.
 
-* New privileges `INSTALL_QUERY` and `EXECUTE_QUERY` are added.
+* New privileges `INSTALL_QUERY`, `EXECUTE_QUERY` and `OWNERSHIP` are added.
+
+[NOTE]
+====
+`OWNERSHIP` is a special privilege introduced since 4.1.0. It should be granted to the user created this query automatically.
```
Comment (victorleeTG) [inline modules/user-access/pages/fine-grained-query-privileges.adoc]:
```suggestion
`OWNERSHIP` is a special privilege introduced in version 4.1.0.
It is granted automatically tp the user who creates a query.
```

File: modules/user-access/pages/fine-grained-query-privileges.adoc [hunk 2]
```diff
@@ -8,11 +8,25 @@ With fine-grained query privileges, following changes will take effects:
...
-* New privileges `INSTALL_QUERY` and `EXECUTE_QUERY` are added.
+* New privileges `INSTALL_QUERY`, `EXECUTE_QUERY` and `OWNERSHIP` are added.
+
+[NOTE]
+====
+`OWNERSHIP` is a special privilege introduced since 4.1.0. It should be granted to the user created this query automatically. 
+
+The entity (a user or a role) who owns a query holds all privileges on a query. That means, this entity can read, update, drop, install and execute this query.
+
+At the same time, this entity can grant the query privileges to other entities, including the `OWNERSHIP` privilege. Once the `OWNERSHIP` privilege is granted to another entity, the previous owner should lose `OWNERSHIP` privilege on this query.
```
Comment (victorleeTG) [inline modules/user-access/pages/fine-grained-query-privileges.adoc]:
```suggestion
An entity with `OWNERSHIP` privilege (e.g., the creator) on a query can grant privileges to other entities, including the `OWNERSHIP` privilege. Once the `OWNERSHIP` privilege is granted to another entity, the previous owner loses `OWNERSHIP` privilege on this query.
[NOTE]
====
Each object can have only one explicit owner entity.  
====
```

Comment (victorleeTG) [reply modules/user-access/pages/fine-grained-query-privileges.adoc]: @jue-yuan: "Should" is the wrong word.
"Should" means "it is the right or correct way".  It also implies that the wrong or incorrect way could happen, so it introduces uncertainty.  We do not want uncertainty about whether TigerGraph implements the language rules or not.

File: modules/user-access/pages/fine-grained-query-privileges.adoc [hunk 3]
```diff
@@ -8,11 +8,25 @@ With fine-grained query privileges, following changes will take effects:
...
+
+Users with the `superuser` role should hold the `OWNERSHIP` privilege on all queries in all graphs implicitly.
+
+Users with the `admin` role in one graph should hold the `OWNERSHIP` privilege on all queries in this graph implicitly.
```
Comment (victorleeTG) [inline modules/user-access/pages/fine-grained-query-privileges.adoc]:
```suggestion
Users with the `superuser` role hold the `OWNERSHIP` privilege on all queries in all graphs implicitly.

Users with the `admin` role in one graph hold the `OWNERSHIP` privilege on all queries in this graph implicitly.
```

File: modules/user-access/pages/fine-grained-query-privileges.adoc [hunk 4]
```diff
@@ -8,11 +8,25 @@ With fine-grained query privileges, following changes will take effects:
...
+====
 
 * Query privilege scope changes:
 ** `CREATE_QUERY` privilege can only be granted/revoked at global and graph level.
 ** `READ_QUERY`, `UPDATE_QUERY`, `DROP_QUERY`, `INSTALL_QUERY` and `EXECUTE_QUERY` privileges can only be granted/revoked at individual query level.
+** `OWNERSHIP` privilege can only be granted at individual query level and only be granted to one entity, e.g.: a user or a user-defined role.
```
Comment (victorleeTG) [inline modules/user-access/pages/fine-grained-query-privileges.adoc]:
```suggestion
** `OWNERSHIP` privilege can only be granted at the individual query level and only to one entity, e.g.: a user or a user-defined role.
```

File: modules/user-access/pages/fine-grained-query-privileges.adoc [hunk 5]
```diff
@@ -88,3 +102,28 @@ User: "u1"
         READ_QUERY
         UPDATE_QUERY
 ----
+
+.Ex. Grant `OWNERSHIP` privilege on existing queries q3 in graph `g1` to user `u1`:
```
Comment (victorleeTG) [inline modules/user-access/pages/fine-grained-query-privileges.adoc]:
```suggestion
.Ex. Grant `OWNERSHIP` privilege on existing query q3 in graph `g1` to user `u1`:
```

**Review by victorleeTG — CHANGES_REQUESTED** [review]
Clarification about OWNERSHIP rules; grammar fixes

**Comment by victorleeTG** [comment]
Cancelled because this PR proposed to merge DOC-2231 into DOC-2230.
Because DOC-2230 was already merged to the dev/staging branch, this PR is being replaced with one to merge DOC-2231 to 4.1.o.

**Pull request closed without merging**